<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisionClass</class>
 <widget class="QMainWindow" name="VisionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Vision</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>987</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_6"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAll"/>
    <addaction name="separator"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="separator"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionGetCode"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionSaveAll"/>
   <addaction name="separator"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="separator"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionGetCode"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="New" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/New.png</normaloff>:/Vision/Resources/icon/New.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="Open" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Open.png</normaloff>:/Vision/Resources/icon/Open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="Save" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Save.png</normaloff>:/Vision/Resources/icon/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAll">
   <property name="icon">
    <iconset theme="SaveAll" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/SaveAll.png</normaloff>:/Vision/Resources/icon/SaveAll.png</iconset>
   </property>
   <property name="text">
    <string>保存全部</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="SaveAs" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/SaveAs.png</normaloff>:/Vision/Resources/icon/SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="Close" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Close.png</normaloff>:/Vision/Resources/icon/Close.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="Quit" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Quit.png</normaloff>:/Vision/Resources/icon/Quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="Undo" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Undo.png</normaloff>:/Vision/Resources/icon/Undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="Redo" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Redo.png</normaloff>:/Vision/Resources/icon/Redo.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="Copy" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Copy.png</normaloff>:/Vision/Resources/icon/Copy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="Paste" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Paste.png</normaloff>:/Vision/Resources/icon/Paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="Cut" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Cut.png</normaloff>:/Vision/Resources/icon/Cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="icon">
    <iconset theme="SelectAll" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/SelectAll.png</normaloff>:/Vision/Resources/icon/SelectAll.png</iconset>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGetCode">
   <property name="icon">
    <iconset theme="getCode" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/getCode.png</normaloff>:/Vision/Resources/icon/getCode.png</iconset>
   </property>
   <property name="text">
    <string>复制代码</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionAttribute">
   <property name="icon">
    <iconset theme="Attribute" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Attritube.png</normaloff>:/Vision/Resources/icon/Attritube.png</iconset>
   </property>
   <property name="text">
    <string>属性</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="About" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/About.png</normaloff>:/Vision/Resources/icon/About.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="Delete" resource="Vision.qrc">
     <normaloff>:/Vision/Resources/icon/Delete.png</normaloff>:/Vision/Resources/icon/Delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Vision.qrc"/>
 </resources>
 <connections/>
</ui>
